src/domain/courseSearch/Search.tsx

```diff
@@ -279,6 +301,27 @@ const Search: React.FC<Props> = ({ scrollToResultList }) => {
                   value={selectedDivisions}
                 />
               </div>
+              <div>
+                <RangeDropdown
+                  checkboxName="ageLimitValues"
+                  rangeIcon={<IconArrowRight aria-hidden />}
+                  minInputValue={minAgeInput}
+                  minInputLabel={t('courseSearch.search.ageLimitMin')}
+                  minInputStartValue={'0'}
+                  minInputFixedValue={'18'}
+                  maxInputValue={maxAgeInput}
+                  maxInputLabel={t('courseSearch.search.ageLimitMax')}
+                  maxInputEndValue={'100'}
+                  name="ageLimitValues"
+                  onChange={handleSetAgeValues}
+                  fixedValuesText={t(
+                    'courseSearch.search.showOnlyAdultCourses'
+                  )}
+                  showFixedValuesText={true}
+                  title={t('courseSearch.search.ageLimitValues')}
+                  value={[minAgeInput, maxAgeInput]}
```

Comment:
Could we use these values [0] and [1] in the component instead of having additional props (minInputValue, maxInputValue)?

Reply:
not sure that i understand the comment